CHANGES.md

```diff
@@ -12,6 +12,7 @@ Change Log
 * Added ability to create partial ellipsoids using both the Entity API and CZML. New ellipsoid geometry properties: `innerRadii`, `minimumClock`, `maximumClock`, `minimumCone`, and `maximumCone`. This affects both `EllipsoidGeometry` and `EllipsoidOutlineGeometry`. See the updated [Sandcastle example](https://cesiumjs.org/Cesium/Apps/Sandcastle/?src=Partial%20Ellipsoids.html&label=Geometries). [#5995](https://github.com/AnalyticalGraphicsInc/cesium/pull/5995)
 * Added `TileMapResourceImageryProvider` and `OpenStreetMapImageryProvider` classes to improve API consistency: [#4812](https://github.com/AnalyticalGraphicsInc/cesium/issues/4812)
 * Added `credit` parameter to `CzmlDataSource`, `GeoJsonDataSource`, `KmlDataSource` and `Model`. [#8173](https://github.com/AnalyticalGraphicsInc/cesium/pull/8173)
+* Added `useBrowserRecommendedResolution` flag to 'Viewer' and 'CesiumWidget'. [8215](https://github.com/AnalyticalGraphicsInc/cesium/issues/8215)
```

Comment:
```suggestion
* Added `useBrowserRecommendedResolution` flag to `Viewer` and `CesiumWidget`. [8215](https://github.com/AnalyticalGraphicsInc/cesium/issues/8215)
```

Reply:
Add a little more context about what the flag does and mention that it replaces the workaround in the 1.61 change list.